### ISSUE TITLE
How to set :rbenv_ruby_version?

### ISSUE THREAD

**Issue by stereodenis** [stas/mina-rbenv-addons#1]
in deploy.rb I set like this
`set :rbenv_ruby_version, '2.2.0'`

mina tasks output:

```
  mina rbenv:install:2.2.0-p0
  mina rbenv:load
  mina rbenv:setup
  mina rbenv:upgrade
```

try

```
mina rbenv:install:2.2.0
mina aborted!
Don't know how to build task 'rbenv:install:2.2.0'
/Users/stereodenis/.rbenv/versions/2.2.0-preview1/lib/ruby/gems/2.2.0/gems/mina-0.3.3/bin/mina:53:in `block (2 levels) in <top (required)>'
/Users/stereodenis/.rbenv/versions/2.2.0-preview1/lib/ruby/gems/2.2.0/gems/mina-0.3.3/bin/mina:26:in `block in <top (required)>'
/Users/stereodenis/.rbenv/versions/2.2.0-preview1/lib/ruby/gems/2.2.0/gems/mina-0.3.3/bin/mina:25:in `instance_eval'
/Users/stereodenis/.rbenv/versions/2.2.0-preview1/lib/ruby/gems/2.2.0/gems/mina-0.3.3/bin/mina:25:in `<top (required)>'
/Users/stereodenis/.rbenv/versions/2.2.0/bin/mina:23:in `load'
/Users/stereodenis/.rbenv/versions/2.2.0/bin/mina:23:in `<main>'
(See full trace by running task with --trace)
```

**Comment by stereodenis** [comment]
oh!
set should be before require?

**Comment by stas** [comment]
Make sure you require the addons in your `deploy.rb` file.

Here's an example file:
https://github.com/Founden/doers/blob/master/config/deploy.rb

**Comment by stereodenis** [comment]
@stas 
`set_default :rbenv_ruby_version, "#{RUBY_VERSION}-p#{RUBY_PATCHLEVEL}"`
how not to use patchlevel?

**Comment by stas** [comment]
Fixed in #2
